framework/src/modules/interoperability/schemas.ts

```diff
@@ -741,3 +741,14 @@ export const genesisInteroperabilitySchema = {
 		},
 	},
 };
+
+export const getRegistrationFeeSchema = {
+	$id: '/modules/interoperability/mainchain/registrationFee',
+	type: 'object',
+	required: ['registrationFee'],
+	properties: {
+		registrationFee: {
+			dataType: 'string',
```

Comment:
```suggestion
			type: 'string',
			format: 'uint64',
```
Since this is JSON, we need to use `type` keyword